tests/test_llm.py

```
@@ -411,6 +411,132 @@ async def test_invoke_command_no_args(self, mock_run: AsyncMock) -> None:
         prompt_idx = call_args.index("-p") + 1
         assert "/review" in call_args[prompt_idx]
 
+    async def test_invoke_command_timeout(self, mock_run: AsyncMock) -> None:
+        """Test that asyncio.timeout context manager enforces timeout."""
+        import asyncio
+
+        from sova.llm.client import invoke_command
+
+        async def slow_operation(*args, **kwargs):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `slow_operation`

Add return type annotation: `None`

(ANN202)